backend/app/main.py

```diff
@@ -171,6 +171,18 @@ async def shatter_parent(
     return result
 
 
+@app.patch(
```

Comment:
I: Use [`app.delete`](https://sqlmodel.tiangolo.com/tutorial/fastapi/delete/)

Reply:
Maybe it should be patch though if we're recreating the partition...

Reply:
I think patch is appropriate here, but no strong opinions. Lmk your preference!